site/.vitepress/sidebar/packages/aa-signers.ts

```diff
@@ -136,5 +136,19 @@ export const aaSignersSidebar: DefaultTheme.SidebarItem = {
       ],
     },
     { text: "Contributing", link: "/contributing" },
+    {
+      text: "Passport Signer",
```

Comment:
nit: can you move this above `Contributing`?

Reply:
fixed!